core/src/main/java/org/pentaho/reporting/platform/plugin/SimpleReportingAction.java

```diff
@@ -112,6 +112,9 @@ public class SimpleReportingAction implements IStreamProcessingAction, IStreamin
 
   public static final String RESERVEDMAPKEY_LINEAGE_ID = "lineage-id";
 
+  public static final String REPORT_EXTERNALIZE_STYLE = "externalize-style";
```

Comment:
I'm fine with the variables. I'm worried about introducing a cyclic build.

The reporting projects - https://github.com/pentaho/pentaho-platform-plugin-reporting and https://github.com/pentaho/pentaho-reporting don't have an API jar.

I just discovered `"useJcr"` is a static constant - https://github.com/pentaho/pentaho-platform-plugin-reporting/blob/9b9b4a15605ce279d3321661f0dc061d3925832f/core/src/main/java/org/pentaho/reporting/platform/plugin/SimpleReportingAction.java#L102